src/main/kotlin/com/numberone/daepiro/domain/disasterContent/service/DisasterContentService.kt

```diff
@@ -0,0 +1,59 @@
+package com.numberone.daepiro.domain.disasterContent.service
+
+import com.numberone.daepiro.domain.dataCollecter.repository.NewsRepository
+import com.numberone.daepiro.domain.disasterContent.dto.response.DisasterContentsResponse
+import com.numberone.daepiro.global.dto.ApiResult
+import com.numberone.daepiro.global.exception.CustomErrorContext.INVALID_DISASTER_CONTENT_SORT_TYPE
+import com.numberone.daepiro.global.exception.CustomException
+import org.springframework.data.domain.PageRequest
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+@Transactional(readOnly = true)
+class DisasterContentService(
+    val newsRepository: NewsRepository
+) {
+    fun getDisasterContents(
+        sortType: String,
```

Comment:
```suggestion
        sortType: SortType,
```

sealed interface 로 sortType 을 정의해도 좋을 것 같네요!